src/form/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Children, FC, useState } from 'react';
```

Comment:
Let's put RadioGroup and Radio together in the same folder.

Reply:
Done. Also updated the doc of `Radio` documentation with an example of `RadioGroup` instead of creating a new doc for the latter.